tests/selenium/pixel_tracking_test.py

```diff
@@ -14,11 +14,17 @@ class PixelTrackingTesting(pbtest.PBSeleniumTest):
 	def test_that_tracker_is_caught(self):
 		TESTING_URL = ("https://gitcdn.link/repo/ablanathtanalba/pixelTrackingTestingResource/master/rsrc.html")
 
+		CLEAR_TRAINED_DATA = (
+		    "chrome.extension.getBackgroundPage().badger.storage.snitch_map.updateObject({})"
```

Comment:
This is OK, but we should probably reuse an existing method (`badger.storage.clearTrackerData()`): https://github.com/EFForg/privacybadger/blob/b52f23d5c9ba03811de9aaad67aa60e1b84292d5/tests/selenium/options_test.py#L54

Reply:
I think you need to visit the options page first, before running this JavaScript.

Reply:
Makes sense, I'll flip that over